start complete
Initiating game:  PandaReach-v3
Experiments starting for:  0
Task Name:  CustomReachEnv1
Traceback (most recent call last):
  File "/home/user/Thesis/trial.py", line 152, in start
    self.run()
  File "/home/user/Thesis/trial.py", line 159, in run
    self.reset()
  File "/home/user/Thesis/trial.py", line 185, in reset
    self.agent.reset()
  File "/home/user/Thesis/agent.py", line 573, in reset
    self.first_state = self.env.reset(seed=self.seedNumber)
AttributeError: 'Agent' object has no attribute 'seedNumber'
Traceback (most recent call last):
  File "/home/user/Thesis/trial.py", line 152, in start
    self.run()
  File "/home/user/Thesis/trial.py", line 159, in run
    self.reset()
  File "/home/user/Thesis/trial.py", line 185, in reset
    self.agent.reset()
  File "/home/user/Thesis/agent.py", line 573, in reset
    self.first_state = self.env.reset(seed=self.seedNumber)
AttributeError: 'Agent' object has no attribute 'seedNumber'